app/models/air_import.rb

```diff
@@ -0,0 +1,78 @@
+require 'securerandom'
+
+class AirImport < MeasurementImport
+  DATA_MESSAGE = "dat"
+
+  belongs_to :user
+
+  has_many :air_logs
+
+  store :status_details, accessors: [ :measurements_added ]
+
+  def import_measurements
+    connection = ActiveRecord::Base.connection.raw_connection
+    name = SecureRandom.uuid
+    connection.prepare name, <<-SQL
+      insert into measurements (user_id, value,       unit, created_at, updated_at, captured_at, measurement_import_id, md5sum, location)
+                         select $1,      measurement, unit, now(),      now(),      captured_at, air_import_id,         md5sum, computed_location
+                           from air_logs
+                          where air_import_id = $2
+    SQL
+    connection.exec_prepared(name, [user_id, id])
```

Comment:
Prepared statements may conflict with a deployment using pgbouncer, so it might be better to not use it explicitly